cockatrice/src/deck_loader.cpp

```diff
@@ -103,6 +103,47 @@ bool DeckLoader::saveToFile(const QString &fileName, FileFormat fmt)
     return result;
 }
 
+struct FormatDeckListForExport
+{
+    QString &mainBoardCards;
```

Comment:
Just wondering; why do these variables start with an ampersand? Are these addresses of the real location?

Reply:
For the struct, those variables are going to be references to the function that calls it. I followed the example in the tapped out class to do that. So you pass the variable to the function you want changed (_mainBoardCards) it's then saved to mainBoardCards `: mainBoardCards(_mainBoardCards),sideBoardCards(_sideBoardCards){}`. Then when you change that variable later in the struct (i.e. in the operator function it changes what you passed in). I'm sure you understand that, but I figured I'd explain it anyway.

Reply:
Ahh, sounds good!